core/src/main/java/com/arcrobotics/ftclib/hardware/ServoEx.java

```diff
@@ -1,23 +1,41 @@
 package com.arcrobotics.ftclib.hardware;
 
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+
 /**
  * An extended servo interface.
  */
 public interface ServoEx extends HardwareDevice {
 
     /**
-     * Rotates the servo a certain number of degrees.
+     * Rotates the servo a certain angle.
+     *
+     * @param angle The desired angle of rotation
+     * @param angleUnit The unit of the angle parameter
+     */
+    void rotateAngle(double angle, AngleUnit angleUnit);
+
+    /**
+     * Rotates the servo a certain angle in degrees.
      *
-     * @param degrees The desired degrees of rotation
+     * @param angle The desired angle of rotation in degrees
      */
-    void rotateDegrees(double degrees);
+    void rotateAngle(double degrees);
```

Comment:
I feel like `rotateBy` is a better name here, since it rotates the current position by a certain number of degrees or radians.

Reply:
Actually, `rotate` should be `rotateBy` and `rotateAngle` should be `rotateByAngle`. It does introduce another breaking change, though.